test/test_undocumented_names.jl

```diff
@@ -0,0 +1,29 @@
+module TestUndocumentedNames
+
+include("preamble.jl")
+
+import PkgWithUndocumentedNames
+import PkgWithoutUndocumentedNames
+
+# Pass
+results = @testtestset begin
+    Aqua.test_undocumented_names(PkgWithoutUndocumentedNames)
```

Comment:
I would expect an `@testset` `begin`-`end` block to be indented, or am I mistaken?

Reply:
There doesn't have to be one, the loop in `runtests.jl` already wraps each file into a `@testset`